Chap_API_Storage.tex

```diff
@@ -23,25 +23,25 @@ \section{Storage support constants}
 
 \begin{constantdesc}
 %
-\declareconstitemProvisional{PMIX_STORAGE_MEDIUM_UNKNOWN}
+\declareconstitemvalueProvisional{PMIX_STORAGE_MEDIUM_UNKNOWN}{0x0000000000000001}
 The storage medium type is unknown.
 %
-\declareconstitemProvisional{PMIX_STORAGE_MEDIUM_TAPE}
+\declareconstitemvalueProvisional{PMIX_STORAGE_MEDIUM_TAPE}{0x0000000000000002}
 The storage system uses tape media.
 %
-\declareconstitemProvisional{PMIX_STORAGE_MEDIUM_HDD}
+\declareconstitemvalueProvisional{PMIX_STORAGE_MEDIUM_HDD}{0x0000000000000004}
 The storage system uses HDDs with traditional SAS, SATA interfaces.
 %
-\declareconstitemProvisional{PMIX_STORAGE_MEDIUM_SSD}
+\declareconstitemvalueProvisional{PMIX_STORAGE_MEDIUM_SSD}{0x0000000000000008}
 The storage system uses SSDs with traditional SAS, SATA interfaces.
 %
-\declareconstitemProvisional{PMIX_STORAGE_MEDIUM_NVME}
+\declareconstitemvalueProvisional{PMIX_STORAGE_MEDIUM_NVME}{0x0000000000000010}
 The storage system uses SSDs with NVMe interface.
 %
-\declareconstitemProvisional{PMIX_STORAGE_MEDIUM_PMEM}
+\declareconstitemvalueProvisional{PMIX_STORAGE_MEDIUM_PMEM}{0x0000000000000020}
 The storage system uses persistent memory.
 %
-\declareconstitemProvisional{PMIX_STORAGE_MEDIUM_RAM}
+\declareconstitemvalueProvisional{PMIX_STORAGE_MEDIUM_RAM}{0x0000000000000040}
 The storage system is volatile (e.g., tmpfs).
 %
 \end{constantdesc}
```

Comment:
The advice to implementers on line 49 is probably unnecessary now.   I think it can be deleted, but maybe someone can verify.